Dockerfile

```diff
@@ -3,18 +3,19 @@ FROM python:3.7-slim-buster as app
 WORKDIR /usr/src/app
 # Install required dependencies and cache test dependencies, so they won't be redownloaded when updating after the code
 # is changed. (We cannot use setup.py here, since it requires other files from this package.)
-COPY requirements.txt requirements-tests.txt ./
+COPY requirements.txt ./
 RUN pip install -r requirements.txt
-RUN pip download -r requirements-tests.txt
 COPY . .
 RUN pip install .
 
 FROM app as tests
 
 # Run linters and tests.
```

Comment:
We don't need this here.

Reply:
you mean that we do not need test stage in dockerfile anymore? Agreed. I'd just remove it in the next step.